src/server.jl

```diff
@@ -239,8 +292,29 @@ function serve_file(
             allow_cors::Bool = false
         )::HTTP.Response
 
+    target = lstrip(req.target, '/')
+    if !isempty(target) && isdir(target) && !endswith(target, '/')
+        return HTTP.Response("""
+            <meta http-equiv="refresh" content="0; URL=/$target/" />
+            """)
```

Comment:
I think this would work, at least most of the time[*], but why not just a `301 Redirect`. I don't really know HTTP.jl, but something like this maybe?

```suggestion
        return HTTP.Response(301, ["Location" => "/$(target)/"], "")
```

It could maybe use something in `body`, although I don't think that's mandatory. GitHub Pages / nginx seem to return something like:

```
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

[*] I wonder if some tools that don't interpret the HTML, but would follow 301 redirects might still not work properly. But I don't think that would be something one would really run into with LiveServer. Should definitely work in browsers.